openage/util/fslike/test.py

```diff
@@ -154,15 +154,42 @@ def test_case_ignoring(root_path, root_dir):
 
     # open it with wrong-case name
     with ignorecase_dir["LeMmE_In"].open("rb") as fil:
+        assert_value(fil.readable(), True)
+        assert_value(fil.writable(), False)
         assert_value(fil.read(), b"pwnt")
 
     # then write it with wrong-case name
     with ignorecase_dir["LeMmE_In"].open("wb") as fil:
-        fil.write(b"changedit")
+        assert_value(fil.readable(), False)
+        assert_value(fil.writable(), True)
+        fil.write(b"just")
 
     # check if changes went to known-name file
     with root_path["lemme_in"].open("rb") as fil:
-        assert_value(fil.read(), b"changedit")
+        assert_value(fil.read(), b"just")
+
+    # append some data
+    with root_path["lemme_in"].open("ab") as fil:
```

Comment:
Putting the various file-opening into `test_case_ignoring()` seems wrong, they should have their own method.